compiler/ras/IgnoreLocale.hpp

```diff
@@ -30,41 +30,11 @@ int toupper_ignore_locale(int c);
 int stricmp_ignore_locale(const char *s1, const char *s2);
 int strnicmp_ignore_locale(const char *s1, const char *s2, size_t n);
 
-#if (J9ZOS390 || AIXPPC || LINUX || OSX)
```

Comment:
Apologies for not seeing the connection, but what does removing all this code have to do with enabling BSD support?

Reply:
The code provides a case insenstive string compare. Ignoring the dead `PILOT` code path, we have the windows case, the main case which covers zos, aix, linux and osx (which uses `strcasecmp`), and the fallback case which provides a simplistic implementation. There a sighned/unsigned warnings in the fallback code (which by default trigger errors). Given that our main case uses `strcasecmp`, which is in the POSIX spec its pretty safe to assume we will have it everywhere (except for windows which we already cover)

Reply:
OK.  So it is related to the BSD support then as opposed to just some cleanup work.